### ISSUE TITLE
Spectral Normalization for Generative Adversarial Networks

### ISSUE THREAD

**Issue by knok** [arXivTimes/arXivTimes#388]
## 一言でいうと

GANの要件であるDiscriminatorのLipschitzs制約を重要視し、Discriminatorの各層にSpectral Normalizationを適用することでGeneratorが精度の高い出力を得られるようになる。

### 論文リンク

* https://drive.google.com/file/d/0B8HZ50DPgR3eSVV6YlF3XzQxSjQ/view

### 著者/所属機関

* Takeru Miyato (Preffered Networks)
* Toshiki Kataoka (Preffered Networks)
* Masanori Koyama (Ritsumeikan University)
* Yuichi Yoshida (National Institute of Informatics)

## 新規性・差分

Discriminatorの各層にSpectrum Normalizationを行うことで、これまで行われていた様々な正規化(batch normalization, weight decay, feature matching, gradient penarty)を必要とすることなく学習が安定し、良い結果が得られる。

## 結果

DCGANのDiscriminatorにSpectrum Normalizationを導入し、CIFAR-10においてInception score 7.41を出し、WGAN-GP, DFM, Cramer GANなど既存のGANより良い結果を出している。

## コメント

PFNet公式のchainer実装あり
* https://github.com/pfnet-research/chainer-gan-lib

**Comment by icoxfog417** [comment]
ご投稿ありがとうございます。以下、レビュー結果です。文章面・内容面の2点の観点から拝見させていただきました。

**文章面**

* 動機の記述：論文がどうした問題を扱っているのかを含めた方が良いと思います。本論文の場合は、GANの学習における不安定性、となると思います。
* 手法の解説：論文の肝となる手法について解説を含めた方が良いと思います。本論文の場合ですと、Spectral Normalizationになると思います。
  * ただ、本論文の場合はちょっと前提事項が多いため、この点について可能な範囲で言及を行った方が読み手の理解を助けると思います。具体的には、本論文はまずGANにおいてDiscriminatorの挙動が性能のカギを握っているという論説があること、かつその挙動を制御するのにLipschitz constant(リプシッツ定数：写像後の空間での距離が元空間の何倍であるかを表す係数？)が正則化項として有用であること、の2点の前提がしかれており、それを踏まえLipschitz constantを制御するためにスペクトルノルム (spectral norm)による制約を加えているように読めます。

**内容面**

* 記載の誤り：解説について誤りのある点はないと思います
* 表現の正確性：「良い結果」は何が良いのか(accuracy/precision/recall)、具体的に書かれていたほうが良いと思います。本研究ではinception scoreを使っています。

**Comment by knok** [comment]
https://www.slideshare.net/DeepLearningJP2016/dl-spectral-norm-regularization-for-improving-the-generalizability-of-deep-learningspectral-normalization-for-gans
日本語解説スライド
